addon/ng2/blueprints/ng2/files/src/tsconfig.json

```diff
@@ -8,9 +8,13 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noImplicitAny": false,
+    "outDir": "../dist/",
     "rootDir": ".",
     "sourceMap": true,
     "sourceRoot": "/",
     "target": "es5"
-  }
+  },
+  "exclude": [
+    "../typings/main.d.ts"
```

Comment:
On the other hand, beta.2 breaks `ng serve` for the same reason, so... a bit of a catch-22 here. You can test this by just upgrading npm angular2 on the test project.